types/locomotive-scroll/locomotive-scroll-tests.ts

```diff
@@ -0,0 +1,16 @@
+import LocomotiveScroll from 'locomotive-scroll';
```

Comment:
```suggestion
import LocomotiveScroll = require('locomotive-scroll');
```
https://github.com/DefinitelyTyped/DefinitelyTyped#esmoduleinteropallowsyntheticdefaultimports